va/va.h

```diff
@@ -454,7 +501,7 @@ typedef enum
      * \brief JPEG encoding attribute. Read-only.
      *
      * This attribute exposes a number of capabilities of the underlying
-     * JPEG implementation. The attribute value is partitioned into fields as defined in the 
+     * JPEG implementation. The attribute value is partitioned into fields as defined in the
```

Comment:
This is just a whitespace removal, but I would prefer to not include this in the FEI patch. Since FEI is a major(non-trivial) extension to VA-API, I recommend a clean patch which shouldn't touch any code blocks other than FEI implementation itself.